plane/src/typed_socket/mod.rs

```diff
@@ -47,7 +53,7 @@ impl<A> From<TrySendError<A>> for TypedSocketError {
     }
 }
 
-impl<A> TypedSocketSender<A> {
+impl<A: Debug> TypedSocketSender<A> {
```

Comment:
I think this isn't necessary because you've derived debug manually?

Reply:
It is to instrument send.